ui-tests/src/test/java/org/openmrs/reference/TransferToWardServiceTest.java

```diff
@@ -11,42 +11,41 @@
 
 import org.junit.After;
 import org.junit.Before;
-import org.junit.Ignore;
 import org.junit.Test;
-import org.openmrs.reference.page.ClinicianFacingPatientDashboardPage;
+import org.junit.experimental.categories.Category;
+import org.openmrs.reference.groups.BuildTests;
+import org.openmrs.reference.page.FindPatientPage;
+import org.openmrs.reference.page.PatientVisitsDashboardPage;
 import org.openmrs.uitestframework.test.TestData;
 
-import static org.junit.Assert.assertTrue;
-
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
 
 public class TransferToWardServiceTest extends ReferenceApplicationTestBase {
 
     private static final String INPATIENT_WARD = "Inpatient Ward";
-
     private static final String ISOLATION_WARD = "Isolation Ward";
-
     private TestData.PatientInfo testPatient;
 
     @Before
     public void createTestData() throws Exception {
         testPatient = createTestPatient();
-        String visitType = TestData.getAVisitType();
-        String visit = new TestData.TestVisit(testPatient.uuid, visitType, getLocationUuid(homePage)).create();
     }
 
-    @Ignore("TODO fix, inpatient_ward doesn't exisit on the page, fix workflow")
     @Test
-    public void transferToWardServiceTest() throws Exception {
-
-        ClinicianFacingPatientDashboardPage patientDashboardPage = homePage.goToActiveVisitPatient();
+    @Category(BuildTests.class)
+    public void transferToWardServiceTest() {
```

Comment:
Refactored the `test logic implementation` to suit the right test workflow for `transferToWardService` test